label_studio/ml_models/models.py

```diff
@@ -26,6 +35,12 @@ class ModelInterface(models.Model):
         'organizations.Organization', on_delete=models.CASCADE, related_name='model_interfaces', null=True
     )
 
+    input_fields = models.JSONField(default=list, validators=[validate_string_list])
+
+    output_classes = models.JSONField(default=list, validators=[validate_string_list])
+
+    associated_projects = models.ManyToManyField('projects.Project', blank=True)
```

Comment:
Why is this allowed to be blank? Seems like we end up having to validate it later too, is this some weirdness with many-to-many fields or something?

Reply:
yes, because we cannot add the manytomany field when we are creating the object and it would error out if it were False. That was the issue I faced and hence I added a check to ensure user provides us the project without it we will not build the modelrun object